src/main/java/domain/Participants.java

```diff
@@ -0,0 +1,42 @@
+package domain;
+
+import constant.Exception;
+
+import java.util.List;
+
+public class Participants {
+
+    public static final int MIN_OF_PARTICIPANTS_COUNT = 2;
+    public static final int MAX_OF_PARTICIPANTS_COUNT = 50;
+    private final List<Person> participants;
+
+    public Participants(List<String> names) {
+        validateCount(names);
+        validateDuplicate(names);
+        this.participants = names.stream()
+                .map(Person::new)
+                .toList();
+    }
+
+    private void validateCount(List<String> names) {
+        if (names.size() < MIN_OF_PARTICIPANTS_COUNT || names.size() > MAX_OF_PARTICIPANTS_COUNT) {
+            throw new IllegalArgumentException(Exception.OUT_OF_RANGE_PARTICIPANTS_COUNT.getExceptionMessage());
+        }
+    }
+
+    private void validateDuplicate(List<String> names) {
+        if (names.stream().distinct().count() != names.size()) {
```

Comment:
스트림은 메서드마다 개행으로 선언형으로 표현하는게 우테코 유행이어서요 ㅎ
```suggestion
        var distinctedNameCount = names.stream()
            .distinct()
            .count();
        if (distinctedNameCount != names.size()) {
```

Reply:
반영하였습니다 감사합니다 ㅎㅎ